src/components/ReferenceViewComponent.js

```diff
@@ -27,14 +34,16 @@ class ReferenceViewComponent {
     pdfContainer: document.querySelector("#pdf-container"),
     contentDiv: document.querySelector("#content-reference-div"),
     sidePageReferenceBtn: document.querySelector("#side-page-reference-btn"),
-    sidePageReferenceContainer: document.createElement("canvas"),
+    sidePageReferenceContainer: document.createElement("div"),
     main: document.querySelector("#main"),
     pdfDoc: null,
     pdfPageNumber: null,
     viewport: null,
```

Comment:
I know this is not your code but this should not be in components.

Reply:
Done